docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -14,41 +14,42 @@ The server node is usually set up to support [multiple private states](../../con
 
 Configure the qlight client using the following command line options:
 
-- `--qlight.client`: This marks the node as a qlight client.
-- `--qlight.client.psi`: This specifies the PSI which this client will support (default = "private").
-- `--qlight.client.serverNode`: The Node ID of the server node.
-- `--qlight.client.serverNodeRPC`: The RPC URL of the server node.
+- [`--qlight.client`](../../reference/cli-syntax.md#qlightclient): This marks the node as a qlight client.
+- [`--qlight.client.psi`](../../reference/cli-syntax.md#qlightclientpsi): This specifies the PSI which this client will support (The default is `private`).
+- [`--qlight.client.serverNode`](../../reference/cli-syntax.md#qlightclientservernode): The Node ID of the server node.
+- [`--qlight.client.serverNodeRPC`](../../reference/cli-syntax.md#qlightclientservernoderpc): The RPC URL of the server node.
 
 If the server node has the RPC API secured using TLS, then the qlight client requires the following:
 
-- `--qlight.client.rpc.tls`: Use TLS when forwarding RPC API calls.
-- `--qlight.client.rpc.tls.insecureskipverify`: Skip TLS verification.
-- `--qlight.client.rpc.tls.cacert`: The qlight client certificate authority.
-- `--qlight.client.rpc.tls.cert`: The qlight client certificate.
-- `--qlight.client.rpc.tls.key`: The qlight client certificate private key.
+- [`--qlight.client.rpc.tls`](../../reference/cli-syntax.md#qlightclientrpctls): Use TLS when forwarding RPC API calls.
+- [`--qlight.client.rpc.tls.insecureskipverify`](../../reference/cli-syntax.md#qlightclientrpctlsinsecureskipverify): Skip TLS verification.
+- [`--qlight.client.rpc.tls.cacert`](../../reference/cli-syntax.md#qlightclientrpctlscacert): The qlight client certificate authority.
+- [`--qlight.client.rpc.tls.cert`](../../reference/cli-syntax.md#qlightclientrpctlscert): The qlight client certificate.
+- [`--qlight.client.rpc.tls.key`](../../reference/cli-syntax.md#qlightclientrpctlskey): The qlight client certificate private key.
 
 If the qlight client node is halted, on restart it resyncs with any blocks that were missed when it was not running.
 
 ## Configure server node
 
 Configure the qlight server using the following command line options:
```

Comment:
Since the options descriptions are simply repeated here, I recommend changing to:

```suggestion
Configure the qlight server using the [`--qlight.server.*`](../../reference/cli-syntax.md#qlightserver) command line options.
```

And deleting the bulleted list.